modules/VersionInfo.cmake

```diff
@@ -21,7 +21,11 @@ else()
 	set( BUILD_VER 0 )
 endif()
 if(NOT BUILD_DATE)
-	string(TIMESTAMP BUILD_DATE "%d.%m.%Y")
+	if (DEFINED ENV{SOURCE_DATE_EPOCH})
+		execute_process(COMMAND "date" "-u" "-d" "@$ENV{SOURCE_DATE_EPOCH}" "+%d.%m.%Y" OUTPUT_VARIABLE BUILD_DATE OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
Well, there’s a benefit of using `SOURCE_DATE_EPOCH` is that it’s supposed to be a standard way to override build time, so if you use it in your build system scripts once, all of your packages will build reproducibly once they’re updated.

Also, it’s distribution-agnostic, so not only Debian benefits from the change, but also a bunch of other distributions providing reproducible builds. See the [spec](https://reproducible-builds.org/specs/source-date-epoch/) for more details.

I hope this explains the reason well enough.